### PR TITLE
Update <blockquote> example

### PR DESCRIPTION
**Pull request by Elchi3**
Example for https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

### CODE REVIEW

File: live-examples/html-examples/text-content/css/blockquote.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  background: #eee;
+  border-radius: 5px;
+}
+blockquote p {
+	padding: 15px;
+}
+
+blockquote p::before,
+blockquote p::after {
+	content: '\201C';
```
Comment (wbamberg) [inline live-examples/html-examples/text-content/css/blockquote.css]: This character is [directional](https://www.fileformat.info/info/unicode/char/201c/index.htm), so I think you'll want:

```
blockquote p::before {
	content: '\201C';
}

blockquote p::after {
  content: '\201D';
}
```

Comment (wbamberg) [reply live-examples/html-examples/text-content/css/blockquote.css]: ...only with less random indentation, I guess :-)

**Comment by Elchi3** [comment]
Thanks Will! Updated.

**Comment by wbamberg** [comment]
This test failure was already there and will be fixed by https://github.com/mdn/interactive-examples/pull/912.

**Pull request merged**